angular.json

```diff
@@ -1,14 +1,34 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
   "version": 1,
+  "defaultProject": "webFrame",
   "newProjectRoot": "projects",
   "projects": {
     "webFrame": {
       "root": "",
       "sourceRoot": "src",
       "projectType": "application",
       "prefix": "app",
-      "schematics": {},
+      "schematics": {
+        "@schematics/angular:component": {
+          "export": true,
+          "changeDetection": "OnPush",
+          "flat": false,
+          "styleext": "scss"
+        },
+        "@schematics/angular:directive": {
+          "export": true,
+          "flat": false
```

Comment:
What is the intent of flat? Googled it but did not retrieve anything clear.

Reply:
For `flat: false` you get a parent directory with the same name. Otherwise the generated component/directive doesn't have a directory of it's own.